hugolib/taxonomy_test.go

```diff
@@ -352,3 +352,33 @@ categories: ["regular"]
 	b.Assert(dra, qt.IsNil)
 
 }
+
+// See https://github.com/gohugoio/hugo/issues/6222
+// We need to revisit this once we figure out what to do with the
+// draft etc _index pages, but for now we need to avoid the crash.
+func TestTaxonomiesIndexDraft(t *testing.T) {
+	t.Parallel()
+
+	b := newTestSitesBuilder(t)
+	b.WithContent(
+		"categories/_index.md", `---
```

Comment:
This is a **taxonomyTerm** kind page's content file. Do you also want to add a test for a **taxonomy** kind page (`content/categories/foo/_index.md`) which matches the scenario in https://github.com/gohugoio/hugo/issues/6222.

Reply:
No, not now. The fix is really isolated and obvious.